blueflood-http/src/main/java/com/rackspacecloud/blueflood/outputs/handlers/HttpMetricDataQueryServer.java

```diff
@@ -79,6 +80,10 @@ public void startServer() {
                         Executors.newFixedThreadPool(workerThreads)));
         server.setPipelineFactory(new MetricsHttpServerPipelineFactory(router));
         serverChannel =  (ServerChannel) server.bind(new InetSocketAddress(httpQueryHost, httpQueryPort));
+
+        //register the tracker MBean for JMX/jolokia
+        log.info("Registering tracker service");
+        Tracker.getInstance();
     }
```

Comment:
I tested that and it doesn't, not sure why.  It'll though once one of the handler call .track or .trackResponse.

To test:  started BluefloodServiceStarter class, started VisualVM (a JMX tool) and looked for com.rackspacecloud.blueflood.tracker, it's not there.  Then I submitted a request through the ingestion server and got a 200, refresh VisualVM and com.rackspacecloud.blueflood.tracker shows up.

I then added the Tracker.getInstance call to the server and restarted BluefloodServiceStarted (which calls the Server.startServer() methods).  Open up VisualVM and it shows up.

JVM must not call the Tracker private constructor until something reference it in the code, is what I am seeing.